src/types/ui.ts

```diff
@@ -50,6 +51,124 @@ export type OracleQueryUI = {
   formattedReward: string | undefined;
   assertion: boolean | undefined;
 };
+export type BigNumberish = string | number | BigNumber;
```

Comment:
these types describe the interface, basically all the data needed to render elements in the ui according to design, in their most primitive form. I know this doesnt match exactly what you have in the ui components, so this or the other may change slightly, but this gives us a point to start implementing functionality